R/data-objects.R

```diff
@@ -27,21 +27,28 @@
 #' # authentication
 #' iauth()
 #'
-#' # some data
-#' foo <- data.frame(x = c(1, 8, 9), y = c("x", "y", "z"))
+#' # creates a csv file of the iris dataset
+#' library(readr)
+#' write_csv(iris, "iris.csv")
+#'
+#' # delete local file
+#' unlink("iris.csv")
 #'
 #' # store
-#' iput(foo, "foo.rds")
+#' iput(iris, "iris.csv")
 #'
 #' # check if file is stored
 #' ils()
 #'
 #' # retrieve in native R format
-#' iget("foo.rds")
+#' iget("iris.csv")
+#'
+#' # check local files
+#' list.files()
 #' }
 iput <- function(
     x,
-    logical_path,
+    logical_path = x,
```

Comment:
I would suggest:
- rename `x` to `local_path`
- `logical_path = basename(local_path)`

(I will take care of the documentation part later)

Reply:
I would also rename `x` in `local_to_irods()`, so it's more clear

Reply:
Good solution.